docs/quality-principles/introduction.md

```diff
@@ -0,0 +1,44 @@
+# Dasharo Quality Principles
+
+## Secure Delivery
+
+Each Dasharo release is secured cryptographically, ensuring with RSA 4096 keys,
+authenticity of binary SHA-256 integrity checksum. The three-step key
+verification assuming that your client receives a clean file without any
+third-party modifications. Integrity and signature verification procedure is
```

Comment:
Integrity and signature verification procedure **- are** included **,** so that the key compliance can be easily checked.

Reply:
fixed